modules/boilerplate/getSurveys.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+const _ = require('lodash');
+const models = require('../../models/index');
+
+// @TODO fetching new surveys requires an app restart... do we care?
```

Comment:
Yes we do care – will change this!

Reply:
... would it be better to check for new/updated surveys every x minutes, or to simply make the query on every pageload? Or stick to as-is right now (eg. they're retrieved at app start time)?